src/OSPSuite.Core/Domain/Descriptors/EntityDescriptor.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace OSPSuite.Core.Domain.Descriptors
+{
+   public class EntityDescriptor
+   {
+      public Tags Tags { get; set; }
```

Comment:
This could be extended to be much more powerful (for example entity type if we want to filter by type etc..)